app/views/resturants/index.html.erb

```diff
@@ -1,78 +1,14 @@
 <p id="notice"><%= notice %></p>
-
 <div class="container">
   <div class="row">
-    <div class="col-md-6" id="filters">
-      <table>
-      <thead>
-        <th>Accepts 10bis:</th>
-        <th>Genre:</th>
-        <th>Rating:</th>
-      </thead>
-      <tr>
-        <td>
-          <input type="checkbox" name="tenbis-filter" value="accepts-tenbis">
-        </td>
-        <td>
-          <select>
-            <% @resturants.each do |resturant| %>
-              <% title = resturant.cuisine.title %>
-              <option value=<%= title %> > <%=title %> </option> 
-            <% end %>
-          </select>
-        </td>
-        <td>
-          <select>
-            <option value="1">1</option>
-            <option value="2">2</option>
-            <option value="3">3</option>
-          </select>
-        </td>
-      </tr>
-    </table>
-  </div>
-</div>
-<div class="row">
-  <div class="col-sm-8">
-    <ul class="list-group row">
-      <% @resturants.each do |resturant| %>
-      <li class="list-group-item col-sm-4" id="cuisine-img">
-        <%= image_tag resturant.cuisine.image %>
-      </li>
-      <li class="col-sm-8" id="resturant-details">
-        <a href=<%= resturant_path(resturant)%>>
-          <div class="col-sm-12">
-            <%= resturant.name %>
-          </div>
-          <div class="col-sm-12">
-            <%= resturant.cuisine.title %>
-          </div>
-          <div class="col-sm-12">
-            Rating:
-            <% resturant.rating.times do %>
-              <i class="fa fa-star"></i>
-            <% end %>
-          </div>
-          <% if resturant.tenbis %>
-            <div class="col-sm-12" id="tenbis">
-              <%= image_tag "10bis.jpg" %>
-            </div>
-          <% end %>
-          <div class="col-sm-12">
-            <%= link_to 'Edit', edit_resturant_path(resturant) %>
-          </div>
-        </a>
-      </li>
-      <% end %>
-    </ul>
+    <div class="col-sm-8 col-md-8">
```

Comment:
It's better to have only one root element and use only one React app main component